src/styles/blocks/card.scss

```diff
@@ -0,0 +1,84 @@
+.product-card {
+  justify-content: center;
+  padding: 32px 16px 16px;
+  width: $card-width;
+  border: 1px solid #f3f3f3;
+  border-radius: $border-radius;
+  box-shadow: 0px 4px 10px rgba(0, 0, 0, 0.3);
+
+  &__image {
+    display: block;
+    margin: 0 auto 40px;
+    width: $card-width - 40px;
```

Comment:
Avoid using calculations in width like `$card-width - 40px`. Instead, consider using padding on `.product-card` or a nested container to control the image size.